provider/schema/column.go

```diff
@@ -95,6 +114,18 @@ func ValueTypeFromString(s string) ValueType {
 		return TypeTimestamp
 	case "uuidarray", "TypeUUIDArray":
 		return TypeUUIDArray
+	case "typeinet", "TypeInet":
```

Comment:
Nit: I guess it is not necessary to check other cases than lower case since you cast it to lower case at the top.

Reply:
fixed shouldn't have the prefix type